flutter_local_notifications/README.md

```diff
@@ -468,7 +468,7 @@ final List<ActiveNotification> activeNotifications =
 ### [Android only] Grouping notifications
```

Comment:
The code shown below was specific to Android and the original comment around iOS was how to handle the lack of grouping support. The changes done here implies that the code also demonstrates grouping through the `threadIdentifier` such that only thing that is missing in the code below is how to show a summary notification on iOS. However, the code hasn't been updated to indicate this. Would it have been possible to update the existing area of the code in the example and this section so that the result on all platforms is as close as they can be? If not, then additional changes would be needed here, possibly even to revert the readme back to what it was before

Reply:
You're right, this has become very confusing, especially as iOS has a native notification summary feature (via categories and `categorySummaryFormat`, `summaryArgument`, `summaryArgumentCount` properties) which is not yet supported by the library. I'll split this section into iOS and Android subsections, as the mechanism is completely different for those two platforms.